### DISCUSSION TITLE
WebpackBin

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#617] [Dicas]
para quem já usou codepen, jsfiddle, e etc, para postar seus snippets de códigos, vocês já conheciam este aqui? [webpackbin.com](https://www.webpackbin.com)

com esse site você pode até testar um SPA simples com Vue.js. saca só o exemplo abaixo:
https://www.webpackbin.com/bins/-KjELUNkEg2hmOzoO7Nu

**Comment by felipefialho** [comment]
Bacana heim!

**Comment by kivervinicius** [comment]
wow! mto top

**Comment by rubenmarcus** [comment]
animal no config vc le os packs do npm q vc quiser... uma coisa só que eu não entendi (to aprendendo ainda) , é o nome "webpackbin", mas nesse link que você postou não vejo nada do webpack

**Comment by simaodeveloper** [comment]
@rubenmarcus na config tem a coluna Loaders, e todo webpack tem um arquivo js de entrada no caso entry.js

**Comment by rubenmarcus** [comment]
@simaodeveloper  sim agora que eu vi ele faz o webpack.config pra vc.. eu baixei o projeto e veio. po esse site pode ser uma boa pra startar projetos e nao ficar configurando o webpack

**Comment by ghost** [comment]
Show, valeu por compartilhar. 👍

**Comment by barraponto** [comment]
Na mesma linha, eu tenho usado o https://codesandbox.io/ pra projetos React.

**Comment by leandross2** [comment]
morreu o site?

**Comment by leandroruel** [comment]
@leandross2 sim, o equivalente agora seria o https://codesandbox.io/
